lib/racecar/runner.rb

```diff
@@ -148,9 +148,11 @@ def producer_config
         "bootstrap.servers"      => config.brokers.join(","),
         "client.id"              => config.client_id,
         "statistics.interval.ms" => config.statistics_interval_ms,
-        "message.timeout.ms"     => config.message_timeout * 1000,
+        "message.timeout.ms"     => config.message_timeout * 1000
```

Comment:
Let's just keep the trailing commas, they make diffs much nicer and avoids some merge conflicts.

Reply:
```suggestion
        "message.timeout.ms"     => config.message_timeout * 1000,
```